actors/builtin/miner/miner_state.go

```diff
@@ -858,34 +858,31 @@ func (st *State) CheckVestedFunds(store adt.Store, currEpoch abi.ChainEpoch) (ab
 	return amountVested, nil
 }
 
-// Unclaimed funds that are not locked -- includes funds used to cover initial pledge requirement
-// does not account for pledge requirements or fee debt.  Always greater than or equal to zero
+// Unclaimed funds that are not locked -- includes free funds and does not
+// account for fee debt.  Always greater than or equal to zero
 func (st *State) GetUnlockedBalance(actorBalance abi.TokenAmount) abi.TokenAmount {
-	unlockedBalance := big.Subtract(actorBalance, st.LockedFunds, st.PreCommitDeposits)
+	unlockedBalance := big.Subtract(actorBalance, st.LockedFunds, st.PreCommitDeposits, st.InitialPledge)
 	Assert(unlockedBalance.GreaterThanEqual(big.Zero()))
 	return unlockedBalance
 }
 
-// Unclaimed funds.  Actor balance - (locked funds, precommit deposit, ip requirement, fee debt)
+// Unclaimed funds.  Actor balance - (locked funds, precommit deposit, initial pledge, fee debt)
 // Can go negative if the miner is in IP debt
 func (st *State) GetAvailableBalance(actorBalance abi.TokenAmount) abi.TokenAmount {
 	unlockedBalance := st.GetUnlockedBalance(actorBalance)
```

Comment:
It's unfortunate that we still end up with both these methods. Can one or the other be removed?

Reply:
Ok, please document it as such. Perhaps we move it to a free function rather than a state method, matching some other only-for-nodes functions?

Reply:
Will do this in 1020